courses/machine_learning/production_ml/3_streaming_data.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>Awesome lab!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/612/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Thanks Benoit! and thanks for the review!</p>